arch/x86/core/intel64/locore.S

```diff
@@ -874,6 +834,50 @@ IRQ(232); IRQ(233); IRQ(234); IRQ(235); IRQ(236); IRQ(237); IRQ(238); IRQ(239)
 IRQ(240); IRQ(241); IRQ(242); IRQ(243); IRQ(244); IRQ(245); IRQ(246); IRQ(247)
 IRQ(248); IRQ(249); IRQ(250); IRQ(251); IRQ(252); IRQ(253); IRQ(254); IRQ(255)
 
+.code16
+.align 4096
+.global x86_ap_start
+x86_ap_start:
```

Comment:
It's unclear why this function is moving, it looks unchanged to me? (Or if it is changed, the deltas are being hidden by code motion)  Can you rework the patch so that the move isn't present?  Or if it does have to move can you put the "motion only" into a separate refactor patch?

Basically: I'm terrified there's something subtly different between two blocks of very subtle assembly code, and the structure of this patch makes that impossible to see.